module-10/10.3/server/jest.config.cjs

```diff
@@ -3,8 +3,8 @@ module.exports = {
     '^.+\\.(js|mjs)$': 'babel-jest',
   },
   testMatch: [
-    '**/__tests__/api/01_dbConnection.test.js',
-    '**/__tests__/api/02_auth.test.js',
-    '**/__tests__/api/03_blog.test.js',
+    '**/__tests__/01_dbConnection.test.js',
+    '**/__tests__/02_auth.test.js',
+    '**/__tests__/03_blog.test.js',
```

Comment:
I think we should be able to just glob-match against anything ending in `*.test.js` in the tests directory